### DISCUSSION TITLE
S2_MSI_L3A_WASP products not avalaible on MDG ?

### DISCUSSION THREAD

**Discussion by SarahMDG** [CS-SI/eodag#445] [Q&A]
Hi everyone,

I am trying to download the S2 L3A products from theia over Madagascar (MDG), all the tiles from 2019-09-01 to 2021-09-01.
The tiles are available throught the theia catalog interface, I know it because I downloaded some of them already manually.

But when I am typing : 
"default_search_criteria = {
    "productType": 'S2_MSI_L3A_WASP',
    "start": "2021-09-01",
    "end": "2019-09-01",
    "geom": {"lonmin": -15.9, "latmin": 46, "lonmax": -17, "latmax": 46.4}
}

location_search_criteria = default_search_criteria.copy()
del location_search_criteria["geom"]
location_search_criteria["locations"] = dict(country="MDG")  
location_search_criteria"

I get : 
"Product type 'S2_MSI_L3A_WASP' is not available with provider 'peps'. Searching it on provider 'theia' instead.
Results found on provider 'theia' is the total number of products available in the searched collection (e.g. SENTINEL2) instead of the total number of products matching the search criteria

Estimated_total_number : 534352"

I am new in exploring the world of the processing chains 🔢 

Thanks for your lighting on this point !
Sarah

**Comment by sbrunato** [comment]
Hello @SarahMDG ,
you have inverted start and end dates. Switch them and you will be able to get S2_MSI_L3A_WASP products!

**Reply by sbrunato** [reply]
And longitude / latitude are also inverted : `{"lonmin": -15.9, "latmin": 46, "lonmax": -17, "latmax": 46.4}` is in the atlantic ocean. For Madagascar, please use `{"latmin": -15.9, "lonmin": 46, "latmax": -17, "lonmax": 46.4}` instead

**Reply by SarahMDG** [reply]
Hello @sbrunato ! Thanks for your help . I did not even notice that I inverted entry parameters. I've tried with the good format and everything works now. 👍🏼
